### PR TITLE
Read org.gnome.settings-daemon.plugins.media-keys screensaver as a list

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: src/service.c [hunk 1]
```diff
@@ -655,9 +655,22 @@ create_switch_section (IndicatorSessionService * self, int profile)
     {
       if (want_accel)
         {
-          gchar * str = g_settings_get_string (p->keybinding_settings, "screensaver");
-          g_menu_item_set_attribute (item, "accel", "s", str);
-          g_free (str);
+            if (ayatana_common_utils_is_mate())
```
Comment (sunweaver) [inline src/service.c:658]: Does this really only apply for MATE? What about Budgie, GNOME (in theory), Unity7? see line 1255 in this file before this change has been applied.

**Review by sunweaver — CHANGES_REQUESTED** [review]
Please check other desktop envs regarding this change. Thanks.

File: src/service.c (hunk 1 above)
Comment (tari01) [reply src/service.c:658]: > Does this really only apply for MATE? What about Budgie, GNOME (in theory), Unity7? see line 1255 in this file before this change has been applied.

Only MATE stores this as a string, all other desktops use `org.gnome.settings-daemon.plugins.media-keys`, which stores this value as a list.

**Review by sunweaver — APPROVED** [review]
Ok, then...

**Pull request merged**